### ISSUE TITLE
PNI - Implement contest ToC page

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#5703]
Comp https://redpen.io/fu065ab5fb8379470d

Copy doc https://docs.google.com/document/d/1YaGrAZZaEPJuy2le_GiWVTshg_yWk6pLuSXLUYexz0Y/edit

Please stick to existing styles used on the guide's about pages. The grid is wider because there is no side nav.

### Todo
- [x] Static page with copy and one image
- [x] URL should be https://foundation.mozilla.org/privacynotincluded/contest
- [x] Page does not appear in nav

###  Design -> Dev Handoff Checklist

- [x] Design has been finalized
- [x] Link to original design ticket - Design contest page #5702
- [x] Link to related ticket
- [x] [Link to Abstract](https://share.goabstract.com/7a3f958d-0082-444b-adf5-2e56c0dc5b72?mode=design&sha=64501a447bd8839f14a194c056b4f294a35a5ec5)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included _(mobile should be straightforward)_
- [x] Name of the Sketch file (Buyers Guide / Contest rules)
- [x] [Assets are included. ](https://drive.google.com/file/d/19awTVP8Mfcox0s0V04iWQhEGkDwjN4Fx/view?usp=sharing)
- [x] Hover state for elements are designed _(use existing styles)_
- [x] Other states for elements are designed (e.g., error state for form)

**Comment by jamilasnell** [comment]
Deadline for PR ready is actual Nov 26 (Thursday)

**Comment by kristinashu** [comment]
This has been approved by legal and is now unblocked!
